packages/react-graphql-universal-provider/src/tests/GraphQLUniversalProvider.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import {InMemoryCache} from 'apollo-cache-inmemory';
-import {ApolloLink} from 'apollo-link';
+import {ApolloLink} from '@apollo/client';
+import {InMemoryCache} from '@apollo/client/cache';
```

Comment:
It looks like this is available from the root entrypoint, no need to reach deep

Reply:
We want to reach deep as client doesn't get exported here: https://github.com/apollographql/apollo-client/blob/main/src/index.ts